spec/features/admin/division_flow_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+feature 'division flow' do
+
+  let(:division) { create(:division) }
+  let(:user) { create(:person, :with_admin_access, :with_password, division: division).user }
```

Comment:
This seems to be repeated a lot. Can we make it its own factory or helper method?